inference-engine/tests/functional/inference_engine/ngraph_reader/logical_not_tests.cpp

```diff
@@ -4,94 +4,80 @@
 
 #include <string>
 #include "ngraph_reader_tests.hpp"
-TEST_F(NGraphReaderTests, DISABLED_ReadLogicalNotNetwork) {
+TEST_F(NGraphReaderTests, ReadLogicalNotNetwork) {
     std::string model = R"V0G0N(
-<net name="Network" version="10">
```

Comment:
Why the original test is removed?

Reply:
The test was not passing at first, after enabling it so I thought that the xml file was a problem so I generated new xml which looked like the one from the documentation, but in the end that was not the case since I received the same error.

Reply:
If original test is passing, we can still use it.

Reply:
So should I remove the new test, and use the old one? Or use both?

Reply:
Use the old one.